admin_interface/templates/admin/change_form.html

```diff
@@ -0,0 +1,68 @@
+{% extends "admin/change_form.html" %}
+{% load static admin_interface_tags %}
+
+
+{% block field_sets %}
+
+{% get_admin_interface_setting "show_fieldsets_as_tabs" as show_fieldsets_as_tabs %}
+{% get_admin_interface_setting "show_inlines_as_tabs" as show_inlines_as_tabs %}
+
+
+    <div class="tab">
+
+      {% if show_fieldsets_as_tabs %}
+        {% for fieldset in adminform %}
+        <button type="button" class="tablinks {{ forloop.counter0|default:"active" }}" onclick="openTab(event, '{{fieldset.name}}')">
+            {{ fieldset.name|default_if_none:opts.verbose_name|capfirst}}
+        </button>
+        {% endfor %}
+      {% else %}
+        <button type="button" class="tablinks active" onclick="openTab(event, 'general')">
+          {{ opts.verbose_name|capfirst }}
+        </button>
+      {% endif %}
+
+     {% if show_inlines_as_tabs %}
+        {% for inline_admin_formset in inline_admin_formsets %}
+        <button type="button" class="tablinks" onclick="openTab(event, '{{inline_admin_formset.opts.verbose_name_plural|capfirst}}')">
+            {{inline_admin_formset.opts.verbose_name_plural|capfirst}}
+        </button>
+        {% endfor %}
+      {% endif %}
+    </div>
+
+    {% if show_fieldsets_as_tabs %}
+      {% for fieldset in adminform %}
+      <div id="{{fieldset.name}}" class="tabcontent {{ forloop.counter0|default:"active" }}">
+          {% include "admin/includes/headerless_fieldset.html" %}
+      </div>
+      {% endfor %}
+    {% else %}
+      <div id="general" class="tabcontent active">
+        {% for fieldset in adminform %}
+          {% include "admin/includes/fieldset.html" %}
+        {% endfor %}
+      </div>
+    {% endif %}
+
+    {% for inline_admin_formset in inline_admin_formsets  %}
+    <div id="{{inline_admin_formset.opts.verbose_name_plural|capfirst}}" class="tabcontent">
+      {% get_inline_template inline_admin_formset.opts.template as inline_template %}
+      {% include inline_template %}
+    </div>
+    {% endfor %}
+    <script
+      type="text/javascript"
+      id="tabbed-changeform-script"
+      src="{% static "admin_interface/js/tabbed_changeform.js" %}"
+    >
+    </script>
+
+{% endblock %}
+
+{% block inline_field_sets %}
```

Comment:
Prefix also block names with `admin_interface_` prefix, it must be easy for everyone to recognize django default admin blocks from the django-admin-interface ones.

Reply:
Here, the django default block is explicitly overridden. To my understanding, that's what allows `{{block.super}}` to work. 

Do you want to set the default `inline_field_sets` block as empty and create a block local to _admin-interface_ ? I didn't go down that path as it'll require this project to manually keep up if any changes are made in the base template.

Reply:
Sorry, I though that it was a block created by you, as not said.